pkg/server/handlers.go

```diff
@@ -30,6 +35,64 @@ func HostsHandler(w http.ResponseWriter, r *http.Request) {
 	RespondWithJson(w, hostList)
 }
 
+type InstallConfig struct {
+	ClusterName string `json:"clusterName"`
+	DNSDomain   string `json:"DNSDomain"`
+	PullSecret  string `json:"pullSecret"`
+}
+
+type DefineClusterCredentials struct {
+	ClusterName string `json:"clusterName"`
+	DNSDomain   string `json:"DNSDomain"`
+	Username    string `json:"username"`
+	Password    string `json:"password"`
+}
```

Comment:
This is a bit confusing, as it conflates the ClusterName/Domain with the credentials required to get the pull secret (which aren't the credentials needed to access the cluster?)